src/cdk/testing/testbed/fake-events/event-objects.ts

```diff
@@ -46,7 +46,9 @@ export function createMouseEvent(
 
   // IE won't set `defaultPrevented` on synthetic events so we need to do it manually.
   event.preventDefault = function() {
-    defineReadonlyEventProperty(event, 'defaultPrevented', true);
+    if(!event.defaultPrevented) {
```

Comment:
Would it make sense to also move the call to `originalPreventDefault` up? That way non-IE browsers will never need to call `defineReadonlyEventProperty`

```ts
const result = originalPreventDefault();
if(!event.defaultPrevented) {
  defineReadonlyEventProperty(event, 'defaultPrevented', true);
}
return result;
```

Reply:
@mmalerba yes. that sounds reasonable. will update the PR